### PR TITLE
Refactorings & reformats

### PR DESCRIPTION
**Pull request by MaxGraey**

### CODE REVIEW

File: assembly/duration.ts [hunk 1]
```diff
@@ -204,12 +211,15 @@ export class Duration {
 
   add<T = DurationLike>(durationToAdd: T, relativeTo: PlainDateTime | null = null): Duration {
     const duration = Duration.from(durationToAdd);
-    const largestUnit = larger(this.largestDurationUnit, duration.largestDurationUnit);
-  
+    const largestUnit = min<i32>(this.largestDurationUnit, duration.largestDurationUnit);
```
Comment (SebastianSpeitel) [inline assembly/duration.ts]: Would `min<TimeComponent>` make it more or less clear?

Comment (MaxGraey) [reply assembly/duration.ts]: what do you mean? Just use `min(this.largestDurationUnit, duration.largestDurationUnit)`?

Comment (SebastianSpeitel) [reply assembly/duration.ts]: That's even better

**Comment by ColinEberhardt** [comment]
Thanks @MaxGraey - all looks good.

Would be great to be able to use prettier in order to auto-format - ref https://github.com/AssemblyScript/assemblyscript/issues/1738 :-)

**Pull request merged**